src/Esfa.Roatp.ApplicationServices/Models/CsvReport/Provider.cs

```diff
@@ -11,6 +11,9 @@ public class Provider
         public bool ParentCompanyGuarantee { get; set; }
 
         public bool NewOrganisationWithoutFinancialTrackRecord { get; set; }
+
         public string StartDate { get; set; }
+
+        public string ProviderNotCurrentlyStartingNewApprentices { get; set; }
```

Comment:
I think this should be a bool in the index as it can only have 2 states and the property name indicates bool.